{
  "cveId": "CVE-2025-55553",
  "eventName": "New CVE Received",
  "cveChangeId": "624EF95D-B7DE-432C-B04F-3F2A3528D69C",
  "sourceIdentifier": "cve@mitre.org",
  "created": "2025-09-25T16:15:34.460",
  "details": [
    {
      "action": "Added",
      "type": "Description",
      "newValue": "A syntax error in the component proxy_tensor.py of pytorch v2.7.0 allows attackers to cause a Denial of Service (DoS)."
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://gist.github.com/shaoyuyoung/0e7d2a586297ae9c8ed14d8706749efc"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://github.com/pytorch/pytorch/issues/151432"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://github.com/pytorch/pytorch/pull/154645"
    }
  ]
}